app/view/button/ShowMeasureToolsPanelController.js

```diff
@@ -0,0 +1,145 @@
+Ext.define('MoMo.client.view.button.ShowMeasureToolsPanelController', {
+    extend: 'Ext.app.ViewController',
+
+    requires: [
+    ],
+
+    alias: 'controller.button.showmeasuretoolspanel',
+
+    /**
+     * Placeholder for the measurement tools panel
+     */
+    btnPanel: null,
+
+    /**
+    *
+    */
+    onToggle: function(btn, pressed){
+        var me = this;
+        if (pressed){
+            me.showMeasureToolsPanel();
+        } else {
+            me.hideMeasureToolsPanel();
+            me.deactivateMeasureTools();
+        }
+    },
+
+    /**
+     * Creates a panel containing two buttons for distance and area measurements
+     * The position of the panel will be computed dynamically (s. method
+     * #computePosition below).
+     */
+    createMeasurementButtonsPanel: function() {
+
+        var me = this;
+
+        var parentBtn = me.getView().getEl();
+
+        var position = me.computePosition(parentBtn);
+
+        var btnPanel = Ext.create('Ext.panel.Panel', {
+            name: 'measurement-buttons-panel',
+            layout: {
+                type: 'hbox',
+                pack: 'right'
+            },
+            width: 100,
+            bodyStyle: {
+                background: 'transparent'
+            },
+            style: {
+                'top': position.top,
+                'right': position.right
+            },
+            defaults: {
+                style: {
+                    margin: '0 5px 0 5px'
+                },
+                xtype: 'basigx-button-measure',
+                toggleGroup: 'measure',
+                showMeasureInfoOnClickedPoints: true,
+                listeners: {
+                    afterrender: function(btn){
+                        btn.setBind({
+                            text: '{text}'
+                        });
+                    }
+                }
+            },
+            items: [{
+                measureType: 'line',
+                glyph: 'xf201@FontAwesome'
+            }, {
+                measureType: 'polygon',
+                glyph: 'xf1fe@FontAwesome'
+            }]
+        });
+
+        return btnPanel;
+    },
+
+    /**
+     * Shows a measurement tools panel on call button toggle.
+     */
+    showMeasureToolsPanel: function() {
+        var me = this;
+        if (!me.btnPanel) {
+            me.btnPanel = me.createMeasurementButtonsPanel();
+            var cont = Ext.ComponentQuery
+                .query('container[name="Map Container"]')[0];
```

Comment:
The selector is probably a bit fragile. But OK for the moment. A `// TODO` would be good, I guess.